src/commands/Tools/create-emoji.ts

```diff
@@ -0,0 +1,43 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { PermissionLevels } from '#lib/types/Enums';
+import { fetch, FetchResultTypes } from '#utils/util';
+import { ApplyOptions } from '@sapphire/decorators';
+import { EmojiRegex } from '@sapphire/discord.js-utilities';
+import { Args } from '@sapphire/framework';
+import { Message } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	aliases: ['add-emoji'],
+	generateDashLessAliases: true,
+	cooldown: 10,
+	permissionLevel: PermissionLevels.Moderator,
+	permissions: ['MANAGE_EMOJIS'],
+	description: LanguageKeys.Commands.Tools.CreateEmojiDescription,
+	extendedHelp: LanguageKeys.Commands.Tools.CreateEmojiExtended
+})
+export class UserCommand extends SkyraCommand {
+	public async run(message: Message, args: SkyraCommand.Args) {
+		const emojiData = await args.pick(UserCommand.emojiResolver);
+
+		try {
+			const emojiBuffer = await fetch(`https://cdn.discordapp.com/emojis/${emojiData.id}.png`, FetchResultTypes.Buffer);
+			const emoji = await message.guild!.emojis.create(emojiBuffer, emojiData.name);
```

Comment:
`emoji.create` takes a Buffer resolvable, which can be a URL. You don't even need to give the name of you just pass the URL if i remember correctly.

Reply:
I can use a URL directly but a name is still required.

Reply:
In that case `message.guild.emojis.resolve(EmojiID).name` would still be better than fetching the image.